src/ducks/classifications.js

```diff
@@ -37,20 +37,23 @@ const classificationReducer = (state = initialState, action) => {
 
 const createClassification = () => {
   return (dispatch, getState) => {
-    let version = "";
+    let workflow_version = '';
     if (getState().workflow.data) {
-      version = getState().workflow.data.version;
+      workflow_version = getState().workflow.data.version;
     }
-    const dimensions = [];
+    
+    let subject = getState().subject;
```

Comment:
Maybe I'm missing something. It doesn't look like this variable is used anywhere below.

Reply:
Thanks for catching this - the error is larger than it appears. I seem to have lost the changes I made for classifications.js in a stash/pop mishap, so I'll need to put back in the code that was lost. Updates incoming.